src/common/error/errorHelper.ts

```diff
@@ -25,7 +25,7 @@ export class ErrorHelper {
     }
 
     public static getNestedWarning(innerError: Error, message: string, ...optionalArgs: any[]): NestedError {
-        return new NestedError(-1, message, innerError, InternalErrorLevel.Warning);
+        return new NestedError(-1, message, innerError, null /* extras */, InternalErrorLevel.Warning);
```

Comment:
This kind of stuff keeps happening to me too :(. We need to find some tslint rule or some way of making the previous code not "compile" any more.